docs/dev/check_references.md

```diff
@@ -99,6 +111,76 @@ The complete list of mandatory and optional attributes for the `manifest.json` f
+|------------------------------------------------------|-------------------------------------------------------------------------|-------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---|
+| `app_id`                                             | String                                                                  | Mandatory                                  | Globally unique name of the offering. Should follow the same standards and limitations of `integration_id` as it is defined today.               |   |
+| `app_uuid`                                           | UUID                                                                    | Mandatory                                  | Globally unique UUID for this application.                                                                                                                                                   |   |
+| `assets`                                             | Object                                                                  | Mandatory                                  | Object containing any Datadog installable entity.                                                                                                |   |
+| `assets[dashboards]`                                 | Object                                                                  | Mandatory, but can be `{ }`                | Out-of-the-box dashboards associated with this offering.                                                                                                                                                                                                       |   |
+| `assets[dashboards["dashboard_short_name"]]`         | String                                                                  | Mandatory                                  | Key/value pairs of any out-of-the-box dashboards. The key is the globally unique short name of the dashboard and the value is the relative path to the dashboard's JSON definition in relation to this manifest.                                      |   |
+| `assets[integration]`                                | Object                                                                  | Mandatory, but can be `{ }`                | Object containing information about the integration.                                                                                                                                                                                                       |   |
```

Comment:
All root "objects" under "assets" is actually `Optional` instead of `{}`

Ex 
* assets->dashboards
* assets->integration
are both `optional`. 

Anything under those keys though are good as they are (i.e.. `assets->integration->configuration`)